File: answers/leetcode/Remove Duplicates from Sorted Array/Remove Duplicates from Sorted Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """

        origin = min(1, len(nums))
        for i in range(1, len(nums)):
            if nums[i] != nums[i - 1]:
                nums[origin] = nums[i]
                origin += 1
        return origin

File: answers/leetcode/Remove Duplicates from Sorted Array/Remove Duplicates from Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int current = 0;
        for (int num : nums) {
            if (num != nums[current]) {
                current += 1;
                nums[current] = num;
            }
        }
        return current + 1;
    }
}